src/app/space/create/deployments/services/deployments.service.ts

```diff
@@ -99,6 +103,21 @@ export interface TimeseriesData {
   net_rx: NetworkReceivedSeries;
 }
 
+export interface MultiTimeseriesData {
+  cores: CoresSeries[];
+  memory: MemorySeries[];
+  net_tx: NetworkSentSeries[];
+  net_rx: NetworkReceivedSeries[];
+  start: number;
+  end: number;
+}
+
+export interface TimeConstrainedStats {
+  cpu: { data: { used: number, quota: number }, timestamp: number }[];
```

Comment:
`{ used: number, quota: number }` is a duplicate of the Stat interface, so this could be written `cpu: { data: CpuStat, timestamp: number }[]`.

Reply:
The whole array type here could also be extracted out into another interface so it's written as `cpu: TimeConstrainedCpuStats` for example, which would save some typing later on too.